barretenberg/scripts/find_github_issues.sh

```diff
@@ -0,0 +1,6 @@
+RESULT=../issues
+> $RESULT
```

Comment:
why define RESULT=../issues then use ../issues? why the > $RESULT if we are appending below?